### ISSUE TITLE
Cenário Atual do Cliente e do Negócio

### ISSUE THREAD

**Issue by marsicanogeorge** [mdsreq-fga-unb/2024.2-T03-PetShopGuara#3]
Todas as causas do problema serão resolvidas com uma solução de software?
Se sim, ok.
Caso contrário, informe o que será resolvido.

**Comment by AlexandreLJr** [comment]
Sim, vamos solucionar todas os problemas com nossa solução. Alteramos no gitpages.
